### DISCUSSION TITLE
Ciabatta ZigBee con misurazione energia

### DISCUSSION THREAD

**Discussion by H4Z4RD-H42** [indomus/forum#2289] [Hardware & Firmware]
Ho bisogno di una ciabatta a 3 posti compatibile ZigBee che monitora i consumi e integrabile su HA. Ne ho trovate diverse su Amazon ma se poi vado sul sito di ZigBee o ZigBee2Mqtt, tra i dispositivi supportati non le trovo. Non capisco quindi se effettivamente la ciabatta è supportata o se è una mancanza dei siti... Quelle che ho visto sono della Vimar, della Oneplug e Meross.

**Comment by indomus** [comment]
ma la vera domanda è: perché vi fissate con zigbee?

**Comment by H4Z4RD-H42** [comment]
In che senso?

Sono abbastanza nuovo del mondo della domotica, grazie alle guide del vostro sito ho messo in piedi tutto l'ecosistema ZigBee installando tutto l'occorrente sul mio NAS QNAP tramite container Docker ed uso una SONOFF Zigbee 3.0 USB Dongle Plus-E.
Detto ciò, ho dato per scontato che d'ora in avanti avrei dovuto acquistare tutti prodotti compatibili con ZigBee in modo da sfruttarne le potenzialità e gestirli comodamente con HA.

Forse intendi che potrei anche acquistare prodotti non direttamente compatibili con ZigBee, ma integrare l'app proprietaria attraverso le integrazioni di HA? Un po' come faccio già con il termostato Tado o le prese smart della TP-Link? Si ok, ma cosi perdo l'occasione di aggiungere dispositivi in giro per casa che mi fanno anche da ripetitori di segnale e onestamente io ne avrei bisogno come il pane visto che ho problemi a raggiungere l'estremità di casa.

Mi puoi spiegare gentilmente cosa intendi più precisamente? Grazie!

**Reply by indomus** [reply]
Hai fatto tutto più che bene. Il senso però è: devo "fissarmi" con uno standard? Risposta: no. Se ho delle alternative più che efficaci su altri standard (nel tuo caso, per esempio, [questa](https://indomus.it/componenti/recensione-meross-multipresa-intelligente-wi-fi-ciabatta-mss425eeu-r/)), perché devo complicarmi la vita? Tutto ovviamente se non hai problemi di copertura Wi-Fi, certo.

**Reply by H4Z4RD-H42** [reply]
Ok chiaro ho capito e ti ringrazio.
